scoreboard/Makefile

```diff
@@ -0,0 +1,67 @@
+####################################################################################
+####################################################################################
+## Copyright 2022(c) Analog Devices, Inc.
+####################################################################################
+####################################################################################
+
+# All test-bench dependencies except test programs
+SV_DEPS += ../common/sv/utils.svh
+SV_DEPS += ../common/sv/logger_pkg.sv
+SV_DEPS += ../common/sv/reg_accessor.sv
+SV_DEPS += ../common/sv/m_axis_sequencer.sv
+SV_DEPS += ../common/sv/s_axis_sequencer.sv
+SV_DEPS += ../common/sv/m_axi_sequencer.sv
+SV_DEPS += ../common/sv/s_axi_sequencer.sv
+SV_DEPS += ../common/sv/test_harness_env.sv
+SV_DEPS += ../common/sv/adi_peripheral_pkg.sv
+SV_DEPS += ../common/sv/adi_regmap_pkg.sv
+SV_DEPS += ../common/sv/mailbox.sv
+SV_DEPS += ../common/sv/x_monitor.sv
+SV_DEPS += ../common/sv/scoreboard.sv
+SV_DEPS += ../common/sv/dmac_api.sv
+SV_DEPS += ../common/sv/dma_trans.sv
+SV_DEPS += ../common/sv/adi_regmap_dmac_pkg.sv
+SV_DEPS += ../common/sv/data_offload_api.sv
+SV_DEPS += ../common/sv/adi_regmap_data_offload_pkg.sv
+SV_DEPS += environment.sv
+SV_DEPS += system_tb.sv
+
+ENV_DEPS += system_project.tcl
+ENV_DEPS += system_bd.tcl
+ENV_DEPS +=../scripts/adi_sim.tcl
+ENV_DEPS +=../scripts/run_sim.tcl
+
+LIB_DEPS := util_cdc
+LIB_DEPS += util_axis_fifo
+LIB_DEPS += axi_dmac
+LIB_DEPS += data_offload
+LIB_DEPS += util_do_ram
+
+# default test program
+TP := test_program
+
+# config files should have the following format
+#  cfg_<param1>_<param2>.tcl
+CFG_FILES := $(notdir $(wildcard cfgs/cfg*.tcl))
+#$(warning $(CFG_FILES))
+
+# List of tests and configuration combinations that has to be run
+# Format is:  <configuration>:<test name>
+TESTS := $(foreach cfg, $(basename $(CFG_FILES)), $(cfg):$(TP))
+#TESTS += cfg1_mm2mm_default:directed_test
```

Comment:
I know these are just comments, but shouldn't we update them according to the actual available config files? In our case there's only cfg1.

Reply:
This comment is for L51-L54, same for the test usage L61 and L64.

Reply:
This should be left as is until we get a guideline up, listing all of the possibilities. Then we can do a clean-up afterwards.